lib/note-content-editor.tsx

```diff
@@ -938,6 +938,10 @@ class NoteContentEditor extends Component<Props> {
     });
 
     editor.onMouseDown((event: Editor.IEditorMouseEvent) => {
+      if((event.event.ctrlKey && event.event.leftButton) || event.event.rightButton) {
```

Comment:
Will `event.event` always exist? Maybe we should also add check for that.

Reply:
Also maybe a space after the `if`? Not sure what our coding standards are here but I see spaces after other ifs in the file.

Reply:
Hah - `prettier` did fix this once I merged the other PR.

As for `event.event` always existing, I believe it must, and TypeScript didn't think it could be `null` either, so perhaps I'll leave that be 🤔